apispec/ext/marshmallow/swagger.py

```diff
@@ -41,6 +41,32 @@
     marshmallow.fields.List: ('array', None),
 }
 
+CUSTOM_FIELD_MAPPING_ATTR = '__swagger_field_mapping'
+
+
+def map_to_swagger_field(*args):
```

Comment:
Fair point. Is `format` actually required? If I recall the spec correctly, it was optional.